Sources/LLBuildManifest/Tools.swift

```diff
@@ -27,13 +27,13 @@ public protocol ToolProtocol: Codable {
     var outputs: [Node] { get }
 
     /// Write a description of the tool to the given output `stream`.
-    func write(to stream: ManifestToolStream)
+    func write(to stream: inout ManifestToolStream)
```

Comment:
Required after converting `ManifestToolStream` to a `struct` with a `String` buffer instead of using TSC streams.